vtiger_connector_crm/models/res_company.py

```diff
@@ -0,0 +1,62 @@
+# See LICENSE file for full copyright and licensing details.
+
+import json
+from urllib.parse import urlencode
+from urllib.request import Request, urlopen
+
+from odoo import models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    def action_sync_vtiger(self):
```

Comment:
Do we need to this method?Because we can directly use "sync_vtiger_crm" from button

Reply:
Yes, we require this method because it is a commen method for schedular.